### ISSUE TITLE
Function strftime() is deprecated

### ISSUE THREAD

**Issue by paveljanda** [middlewares/access-log#22]
Full error message:

```
PHP message: PHP Deprecated:  Function strftime() is deprecated in /var/www/vendor/middlewares/access-log/src/AccessLogFormats.php on line 270
```

**Comment by paveljanda** [comment]
More info: https://php.watch/versions/8.1/strftime-gmstrftime-deprecated

**Comment by oscarotero** [comment]
thanks!
Fixed in v2.1.1.

**Comment by paveljanda** [comment]
Nice, thanks for fast response!
